consent-ws/src/main/java/org/broadinstitute/consent/http/resources/ConsentResource.java

```diff
@@ -180,4 +187,15 @@ public ConsentResource() {
         this.electionAPI = AbstractElectionAPI.getInstance();
     }
 
+    private void checkValidDUL(Consent rec) {
```

Comment:
It'd be nice to make methods like this less side-effecty and more strongly typed like we do in our Scala codebase. Not a blocker...just something to think about in case new Java standard/non-standard libraries make that more practical now.